test/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "bedrock-vue-address-form-test",
+  "version": "0.0.1-0",
+  "description": "Vue.js address form component for Bedrock Web apps test",
+  "private": true,
+  "main": "index.js",
+  "less": "app.less",
```

Comment:
Is this going to continue being a thing now that we're moving towards scss?

Reply:
No, but this can be remedied in the future. I'll file an issue.